.gitattributes

```diff
@@ -0,0 +1,21 @@
+# Package manager lock files
+Pipfile.lock linguist-generated
+package-lock.json linguist-generated
+
+# Generated GraphQL schemas
+*.graphql linguist-generated
+
+# Generated GraphQL types
+gql-types.ts linguist-generated
+
+# Generated django migration files
+*_auto_*.py linguist-generated
+
+# Generated locale pot files
+*.po linguist-generated
```

Comment:
These are not really generated.

Reply:
How do we want to handle them? Do we want to show the diffs?

Reply:
I'd like to see a diff on GitHub but would explicitly tell git not to attempt a three-way merge.